master/buildbot/test/unit/test_steps_shell.py

```diff
@@ -421,7 +430,11 @@ def test_run_property_no_strip(self):
         self.expectOutcome(result=SUCCESS,
                            state_string="property 'res' set")
         self.expectProperty("res", "\n\nabcdef\n")
-        self.expectLogfile('property changes', r"res: u'\n\nabcdef\n'")
+        if PY3:
```

Comment:
OK, I have removed this particular part of the patch, and will work on this in a separate PR.

The rest of this patch should be good to go in.